src/constants/tokenConfig.ts

```diff
@@ -88,6 +90,32 @@ export const OUTPUT_TOKEN_CONFIG: Record<OutputTokenType, OutputTokenDetails> =
     minWithdrawalAmountRaw: '10000000000000',
     maxWithdrawalAmountRaw: '10000000000000000',
     offrampFeesBasisPoints: 125,
+    supportsClientDomain: true,
+  },
+  ars: {
+    tomlFileUrl: 'https://api.anclap.com/.well-known/stellar.toml',
+    decimals: 12,
+    fiat: {
+      assetIcon: 'ars',
+      symbol: 'ARS',
+    },
+    stellarAsset: {
+      code: {
+        hex: '0x41525300',
+        string: 'ARS\0',
```

Comment:
Should we add the `new Asset()` constructor compatible version to the token config maybe so we don't need to `replace()` the `\0` everywhere? Or we refactor the code to reduce the calls to `replace()`.
```suggestion
        string: 'ARS',
        raw: 'ARS\0',
```

Reply:
Yes, that's good 👍  I see that you changed it from `string` to `stringRaw` back to `string`. I think for consistency it would be better to also call it `stringRaw` and `stringStellar` on the typescript/frontend part even if it means touching some references.